CHANGES.rst

```diff
@@ -27,6 +27,7 @@ mast
 - Removed deprecated ``Observations.get_hst_s3_uris()``, ``Observations.get_hst_s3_uri()``, 
   ``Core.get_token()``, ``Core.enable_s3_hst_dataset()``, ``Core.disable_s3_hst_dataset()`` and 
   variables obstype and silent [#1884]
+- Added case for catching empty observations passed to ``Observations.get_product_list()`` [#1921]
```

Comment:
Super nitpicky:
- Mentioning the raised error is more useful from the users' POV, so I would suggest a rephrasing.

Reply:
Got it! Just pushed a reworded version.